main.ts

```diff
@@ -61,13 +67,31 @@ async function run(): Promise<void> {
       }
     }
 
-    const mingw = architecture === 'i686' ? 'MINGW32' : 'MINGW64'
+    let mingw = ''
+
+    switch (architecture) {
+      case 'i686':
+        mingw = 'MINGW32'
+        break
+      case 'x86_64':
+        mingw = 'MINGW64'
+        break
+      case 'aarch64':
+        mingw = 'CLANGARM64'
+        break
+      default:
+        core.setFailed(`Invalid architecture ${architecture} specified`)
+    }
+
     const msystem = msysMode ? 'MSYS' : mingw
 
     const binPaths = [
       // Set up PATH so that Git for Windows' SDK's `bash.exe`, `prove` and `gcc` are found
       '/usr/bin/core_perl',
       '/usr/bin',
+      // Some binaries aren't available yet in the /clangarm64/bin folder, but Windows 11 ARM64
+      // has support for x64 emulation, so let's add /mingw64/bin as a fallback.
+      ...(msystem === 'CLANGARM64' ? ['/mingw64/bin'] : []),
```

Comment:
I was specifically missing `git.exe` and friends. This fallback will no longer be needed once a first version of Git for Windows ARM64 has been released.